### DISCUSSION TITLE
The length of a single message is 100mb and the subscription client has a large delay. [win10 binary] [performance]

### DISCUSSION THREAD

**Discussion by stan1334** [emqx/emqx#13058] [Q&A] [answered]
test environment:
3 pub clients
6 subclients
windows binary deployment
win10
x64
cpu: 12th Gen Intel(R) Core(TM) i7-12700 2.10 GHz
memory: 16GB

Continuously send 50 pieces of data with a length of 100mb in a single time
The subscription client receives 50 pieces of data and the delay reaches 5 seconds.

100mb is changed to less than 30mb, and the delay is reduced to 1-2 seconds.

How should I optimize if there are often 100mb of data in the actual production environment?
@zmstone
@thalesmg

**Comment by zmstone** [comment]
Hi @stan1334 
We have stopped supporting windows.
Could you run the tests with EMQX version 5 on Linux servers?

**Reply by stan1334** [reply]
i try on dockerimg windows 5.6.1，100mb singleData allways need 1-2Second。
Our production environment is windows binary, I am using 5.3.2， broker and client on one machine。

![image](https://github.com/emqx/emqx/assets/59048253/28b58d54-ecaf-4fdc-b8d0-dbbba33f0402)


![image](https://github.com/emqx/emqx/assets/59048253/22730c91-ea2a-4045-849a-6c74bdfe9728)

@zmstone 
Thank you for your reply. Even though this problem does not exist in the Linux environment, I still cannot solve the problem in Windows.
Is there any other solution in windows?

**Reply by ieQu1** [reply, marked as answer]
Sorry, we dropped support for Windows completely.

1) No one in our team runs Windows on a regular basis, so it's very hard for us to troubleshoot problems on that platform. 
2) Windows builds were never targeted for production use, only for evaluation.
3) Windows users can use WSL.
